pytest_mock.py

```diff
@@ -137,3 +137,84 @@ def mock(mocker):
     warnings.warn('"mock" fixture has been deprecated, use "mocker" instead',
                   DeprecationWarning)
     return mocker
+
+
+_mock_module_patches = []
+_mock_module_originals = {}
+
+
+def assert_wrapper(method, *args, **kwargs):
+    __tracebackhide__ = True
+    try:
+        method(*args, **kwargs)
+    except AssertionError as e:
+        raise AssertionError(*e.args)
+
+
+def wrap_assert_not_called(*args, **kwargs):
+    __tracebackhide__ = True
+    assert_wrapper(_mock_module_originals["assert_not_called"],
```

Comment:
Couldn't you just call the wrapped method directly, and let the `AssertionError` propagate? Like this:

```python
def wrap_assert_not_called(*args, **kwargs):
    __tracebackhide__ = True
    _mock_module_originals["assert_not_called"], *args, **kwargs)
```